features/lwipstack/lwip-sys/arch/lwip_sys_arch.c

```diff
@@ -552,12 +562,12 @@ static sys_thread_data_t thread_pool[SYS_THREAD_POOL_N];
 
 #include <stdbool.h>
 
-#if MBED_CONF_LWIP_USE_MBED_TRACE
 #include "mbed-trace/mbed_trace.h"
 
 void lwip_mbed_tracef_debug(const char *fmt, ...)
 {
     va_list ap;
+#if MBED_CONF_LWIP_USE_MBED_TRACE
```

Comment:
Making a note here that this'll probably result in a code size increase.

Reply:
Oh, I didn't intend to make that change. I'll look into that.